src/storage/internal/lookup_tables.py

```diff
@@ -13,7 +13,7 @@ def insert_lookup(lookup_id: str, lookup: dict) -> None:
         lookup_table_collection.insert_one(lookup)
     except DuplicateKeyError:
         raise BadRequestException(
-            f"A lookup table with name '{lookup['name']}' already exists."
+            f"A lookup table with name '{lookup['_id']}' already exists."
```

Comment:
```suggestion
            f"A lookup table with id '{lookup['_id']}' already exists."
```
Suggestion

Reply:
For the user, the lookup is refered to by a name.
It just so happens to be that we use that name as a primaryKey as well. No need to inform the user of that.